packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -386,6 +389,33 @@ export class SceneTree {
     });
   }
 
+  /**
+   * Performs an API call that will select the items with indexes between the given bounds.
+   *
+   * @param lowerBound The smaller index of the set of rows to select.
+   * @param upperBound The larger index of the set of rows to select.
+   * @param viewer The viewer to select in.
+   */
+  @Method()
+  public async performMultiSelection(
```

Comment:
Naming nit, but maybe call this `selectRange` so it shows up in docs near our other select methods.